treeherder/model/models.py

```diff
@@ -210,14 +210,14 @@ def __str__(self):
 
 @python_2_unicode_compatible
 class Bugscache(models.Model):
-    id = models.AutoField(primary_key=True)
-    status = models.CharField(max_length=64, blank=True, db_index=True)
+    id = models.PositiveIntegerField(primary_key=True)
```

Comment:
Ie: omitting `id` on . create() should be an error, not silently create a row with an arbitrary bug number.